packages/ListBox/src/components/Filter/helpers.js

```diff
@@ -21,6 +29,15 @@ export const filter = ({ state, textSearchValue }) => {
       return false;
     });
 
+    const filteredOptions = filteredContent.filter((key, index) => {
+      const nextKey = filteredContent[index + 1];
+      if (isDivider(key)) {
+        if (!nextKey) return false;
```

Comment:
I believe for readiablity we like to have { } after if statement so
```suggestion
         if(!nextKey) {
           return false;
         }
```